packages/contracts-bedrock/src/L1/OPContractsManager.sol

```diff
@@ -198,9 +198,9 @@ contract OPContractsManager is ISemver {
     function deploy(DeployInput calldata _input) external returns (DeployOutput memory) {
         assertValidInputs(_input);
         uint256 l2ChainId = _input.l2ChainId;
-        // The salt for a non-proxy contract is a function of the chain ID and the salt mixer.
         string memory saltMixer = _input.saltMixer;
-        bytes32 salt = keccak256(abi.encode(l2ChainId, saltMixer));
+        // The salt for a non-proxy contract is a function of the chain ID and the salt mixer.
+        bytes32 singleContractSalt = computeSingleContractSalt(l2ChainId, saltMixer);
```

Comment:
nit to match the name in the comment, which I think is more clear anyway
```suggestion
        bytes32 nonProxiedSalt = computeSingleContractSalt(l2ChainId, saltMixer);
```

Reply:
good call, added in: https://github.com/ethereum-optimism/optimism/pull/13273/commits/b4d29d97b03b6ba6dc0a4ec77a146dbec7a0f1d2